scripts/build_test.py

```diff
@@ -24,7 +33,79 @@
 class BuildTests(test_utils.GenericTestBase):
     """Test the build methods."""
 
+    def test_minify(self):
+        """Tests _minify with an invalid filepath."""
+        # pylint: disable=protected-access
+        with self.assertRaises(subprocess.CalledProcessError) as calledProcess:
+            build._minify(
+                'path/to/input.js', 'path/to/output.js')
+        # pylint: enable=protected-access
+        calledProcessException = calledProcess.exception
+        # returncode is the exit status of the child process.
+        self.assertEqual(calledProcessException.returncode, 1)
+
+    def test_join_files(self, file_count=10):
+        """Determine third_party.js contains the content of the first 10 JS
+        files in /third_party/static.
+        """
+        third_party_js = os.path.join(
+            build.THIRD_PARTY_GENERATED_DEV_DIR, 'js', 'third_party.js')
+        # Get all filepaths from manifest.json.
+        dependency_filepaths = build.get_dependencies_filepaths()
+        # Join all JS files in /third_party/static.
+        # pylint: disable=protected-access
+        build._join_files(dependency_filepaths['js'], third_party_js)
+        # pylint: enable=protected-access
+        with open(third_party_js, 'r') as joined_js_file:
+            # Open third_party.js.
+            joined_js_file_content = joined_js_file.read()
+            counter = 0
+            for js_filepaths in dependency_filepaths['js']:
+                # Open first 10 JS files.
+                if counter == file_count:
+                    break
+                with open(js_filepaths, 'r') as js_file:
+                    # Assert that each line is copied over to third_party.js.
+                    for line in js_file:
+                        self.assertIn(line, joined_js_file_content)
+                counter += 1
+        os.remove(third_party_js)
+
+    def test_minify_and_create_sourcemap(self):
+        """Tests _minify_and_create_sourcemap with an invalid filepath."""
+        # pylint: disable=protected-access
+        with self.assertRaises(subprocess.CalledProcessError) as calledProcess:
+            build._minify_and_create_sourcemap(
+                'path/to/input.js', 'path/to/output.js')
```

Comment:
Perhaps include "invalid" in the invalid filepath.

Reply:
Done!